lib/jabbax/document/error_source.ex

```diff
@@ -3,7 +3,16 @@ defmodule Jabbax.Document.ErrorSource do
 
   defstruct [:pointer, :parameter]
 
+  def from_attribute(attributes) when is_list(attributes) do
```

Comment:
This is not a list of attributes, it's a path to the attribute in question:

```suggestion
  def from_attribute(attribute_path) when is_list(attribute_path) do
```